### PR TITLE
Add tabs in MainActivity

### PR DESCRIPTION
**Pull request by aerinpark**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml
Comment (bperdija) [inline app/src/main/AndroidManifest.xml:1]: See if we can default to light mode

**Review by bperdija — CHANGES_REQUESTED** [review]

File: app/src/main/AndroidManifest.xml
Comment (aerinpark) [reply app/src/main/AndroidManifest.xml:1]: Fixed. Now it's in light mode

**Review by bperdija — APPROVED** [review]
Looks great!!! Thank you! :)

**Review by Jramacula — APPROVED** [review]
Looks good! :+1:

**Pull request merged**